renku_notebooks/api/notebooks.py

```diff
@@ -225,6 +227,25 @@ def launch_notebook(
             jsonify({"messages": {"error": f"Cannot find/access image {image}."}}), 404
         )
 
+    # create the PVC if requested
+    pvc_name = ""
+
+    if config.NOTEBOOKS_USE_PERSISTENT_VOLUMES:
+        pvc_name = f"{safe_username}-{make_server_name(namespace, project, branch, commit_sha)}-pvc"
+        pvc = create_pvc(
+            name=pvc_name,
+            username=safe_username,
+            git_namespace=namespace,
+            project_id=gl_project.id,
+            project=project,
+            branch=branch,
+            commit_sha=commit_sha,
+            git_host=urlparse(config.GITLAB_URL).netloc,
+            storage_size=server_options.get("disk_request"),
+            storage_class="temporary",
```

Comment:
So the `temporary` storage class is not available in different providers/k8s clusters. For example `gke` will have only `premium-rwo`, `standard` and `standard-rwo` by default. I had this issue when I was deploying renku on gcp. Jena uses also a `temporary` storage class (but this is adjustable in the `values.yaml` file for renku in the Jena/kg section). So I think that this value should also be specified by the user in the `values.yaml` file and not be hardcoded in here.

Reply:
Also it may be that someone wants to use a fancier storage class backed by ssd or something else for the user sessions.

Reply:
definitely - it was on my todo to make this configurable but I forgot. I guess it will be up to the operator then to figure out how to actually remove the PVs? We could include a template for the "temporary" storage class in the chart so it's created by default if possible?